From ec194da7649ac6b2351f4e9747b9f04cb26cf2e6 Mon Sep 17 00:00:00 2001
From: Casey Connolly <casey.connolly@linaro.org>
Date: Sat, 12 Jun 2021 14:09:57 +0100
Subject: [PATCH 15134/15333] arm64: dts: qcom: sdm845-oneplus: Add haptics
 support

Enable the pmi8998 SPMI haptics driver and configure the play rate for the
hardware.

Signed-off-by: Casey Connolly <casey.connolly@linaro.org>
Signed-off-by: David Heidelberg <david@ixit.cz>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 439779f308e3..cedb02cc3c51 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -666,6 +666,12 @@ led-1 {
 	};
 };
 
+&pmi8998_haptics {
+	qcom,wave-play-rate-us = <4255>;
+
+	status = "okay";
+};
+
 &q6afedai {
 	dai@22 {
 		reg = <QUATERNARY_MI2S_RX>;
-- 
2.55.0

